### PR TITLE
new change data-read-method to Textline

### PR DESCRIPTION
**Pull request by luffywalf**
**Thank you for submitting a pull request! Please provide the following information for code review:**

# Pull Request Summary
new change data-read-method from [from_tensor_slices] to [textline_dataset]
# Test Plan
tested

### CODE REVIEW

File: delta/data/preprocess/base_preparer.py [hunk 1]
```diff
@@ -101,68 +102,80 @@ def prepare_raw_data(self, pre_process_pipeline):
...
       all_texts += text_after
       one_text_after = text_after
+    self.config['data']['{}_data_size'.format(mode)] = len(one_text_after[0])
+    one_label_after = []
+    if not infer_without_label:
+      if self.multi_output:
+        for i in range(self.output_num):
+          label_ds = label[i].batch(self.batch_size)
+          label_iterator = label_ds.make_initializable_iterator()
+          label__after_arr = self.run_dataset(label_iterator, batch_num)
```
Comment (zh794390558) [inline delta/data/preprocess/base_preparer.py]: fix label__after_arr

File: delta/data/task/text_match_task.py [hunk 2]
```diff
@@ -178,7 +177,7 @@ def dataset(self):
         "input_x_len": input_x_len,
         "iterator": iterator,
         "iterator_len": iterator_len,
-        "init_feed_dict": self.init_feed_dict
+        #"init_feed_dict": self.init_feed_dict   #去掉
```
Comment (zh794390558) [inline delta/data/task/text_match_task.py]: do not use Chinese characters

File: delta/data/task/text_seq_label_task.py [hunk 3]
```diff
@@ -49,41 +51,35 @@ def __init__(self, config, mode):
...
-    self.init_feed_dict[label_placeholder] = label
-
-    text_ds = self.load_text_dataset(text_placeholder)
-
+    logging.info("process label ds...")
     if self.infer_without_label:
       data_set = text_ds
     else:
-      label_ds = load_multi_label_dataset(label_placeholder, self.config)
+      label_ds = load_multi_label_dataset(label_ds, self.config)  # TODO one_label 也要改
```
Comment (zh794390558) [inline delta/data/task/text_seq_label_task.py]: same

File: egs/conll2003/pretrain/v1/config/seq_label_elmo.yml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  service:
+    model_path: "egs/conll2003/pretrain/v1/exp_elmo/bilstmcrf_elmo/service"
+    model_version: "1"
+  run_config:
+    tf_random_seed: null
+    allow_soft_placement: true
+    log_device_placement: false
+    intra_op_parallelism_threads: 10
+    inter_op_parallelism_threads: 10
+    allow_growth: true
```
Comment (zh794390558) [inline egs/conll2003/pretrain/v1/config/seq_label_elmo.yml]: what's this `submodule bilm `?

Comment (luffywalf) [reply egs/conll2003/pretrain/v1/config/seq_label_elmo.yml]: run run_elmo.sh,  it will be downloaded then

File: delta/data/task/text_cls_task.py [hunk 5]
```diff
@@ -53,21 +55,19 @@ def __init__(self, config, mode):
...
     ]
+    self.infer_without_label = bool(mode == utils.INFER and self.infer_no_label)
+
     self.prepare()
 
   def generate_data(self):
     """Generate data for offline training."""
+    if self.infer_without_label:
+      column_num = 1
+      text_ds=load_textline_dataset(self.paths_after_pre_process, column_num)
```
Comment (applenob) [inline delta/data/task/text_cls_task.py]: This line seems not formatted, pls check.

File: delta/data/utils/common_utils.py [hunk 6]
```diff
@@ -330,7 +332,7 @@ def load_npy(npy_path, dtype=np.float32):
   return dense_feature
 
 
-def load_one_label_dataset(label_placeholder, config, output_index=None):
+def load_one_label_dataset(label_ds, config, output_index=None):
```
Comment (applenob) [inline delta/data/utils/common_utils.py]: change name to `process_one_label_dataset`

**Pull request merged**